scripts/pool.py

```diff
@@ -28,13 +28,18 @@ def init_click_context(ctx: click.Context, environment: str, role: str, inactive
     ctx.obj['ENVIRONMENT'] = environment
     ctx.obj['FILTER'] = {}
     ctx.obj['FILTER']['ROLE'] = role
+    ctx.obj['FILTER']['POOL'] = pool
     ctx.obj['INACTIVE'] = inactive 
     ctx.obj['DEBUG'] = debug
 
     if ctx.obj['DEBUG']:
         click.echo("## DEBUG: create an oracle ComputeManagementClient for each region in OCI")
     oci_config = oci.config.from_file()
-    for region in hcvlib.oracle_regions():
+    if oracle_region:
```

Comment:
if we are going to change this, maybe we should handle a space-delimited list of regions instead rather than limit the option to a single region or all regions